core-modules/be-core-modules/data-bases/relationalDataBase/models/prod/Answer.js

```diff
@@ -2,18 +2,21 @@ module.exports = function(sequelize, DataTypes) {
   var modelName = 'answer'
 
   const Answer = sequelize.define(modelName, {
-    label: {
+    response: {
```

Comment:
This name is actually the label of the response. Feel free to keep it but I think label is a better choice

Reply:
response is de response that you choise, so, y called it in that way

Reply:
Fixed